src/components/directory-item-selector.js

```diff
@@ -123,14 +127,31 @@ const DirectoryItemSelector = (props) => {
     const fetchDirectory = useCallback(
         (nodeId) => {
             fetchDirectoryContent(nodeId)
-                .then((childrenToBeInserted) => {
-                    // update directory Content
-                    addToDirectory(
-                        nodeId,
-                        childrenToBeInserted.filter((item) =>
-                            contentFilter().has(item.type)
-                        )
+                .then((children) => {
+                    let childrenMatchedTypes = children.filter((item) =>
```

Comment:
const unstead of let cause we don't have a reassignment to childrenMatchedTypes

Reply:
Done